bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/JournalChannel.java

```diff
@@ -248,6 +257,13 @@ private JournalChannel(File journalDirectory, long logId,
         }
     }
 
+    public static void renameJournalFile(File source, File target) throws IOException {
+        if (!source.renameTo(target)) {
```

Comment:
I  think you should check source's value like:
`if (source == null) {
            throw new NullPointerException();
        }`